src/uproot/const.py

```diff
@@ -114,3 +114,9 @@
 ############# other
 
 kStreamedMemberWise = numpy.uint16(1 << 14)
+
+############ RNTuple https://github.com/root-project/root/blob/master/tree/ntuple/v7/doc/specifications.md
+rt_role_leaf = 0
+rt_role_vec = 1
+rt_role_struct = 2
+rt_role_union = 3
```

Comment:
No reason to be so brief:

```suggestion
rntuple_role_leaf = 0
rntuple_role_vec = 1
rntuple_role_struct = 2
rntuple_role_union = 3
```

But then you have to change their usages below. I found a few more examples of literal values that should be replaced with constants.